addons/stopwatch/stopwatch.lua

```diff
@@ -0,0 +1,87 @@
+_addon.name = 'stopwatch'
+_addon.author = 'Patrick Finnigan (Puhfyn@Ragnarok)'
+_addon.version = '1.0'
+_addon.commands = {'sw', 'stopwatch'}
+
+require('logger')
+config = require('config')
+texts = require('texts')
+ 
+defaults = {}
+defaults.pos = {}
+defaults.pos.x = 450
+defaults.pos.y = 0
+defaults.text = {}
+defaults.text.font = 'Arial'
+defaults.text.size = 12
+ 
+settings = config.load(defaults)
+times = texts.new(settings)
+startTime = os.time()
```

Comment:
While not mandatory, it's a convention to use `snake_case`for Windower addon code. This is up to you though, just letting you know.